pkgs/applications/networking/instant-messengers/element/seshat/default.nix

```diff
@@ -0,0 +1,44 @@
+{ rustPlatform, fetchFromGitHub, callPackage, sqlcipher, nodejs-14_x, python3, yarn, fixup_yarn_lock }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "seshat-node";
+  version = "2.3.0";
+
+  src = fetchFromGitHub {
+    owner = "matrix-org";
+    repo = "seshat";
+    rev = version;
+    sha256 = "0zigrz59mhih9asmbbh38z2fg0sii2342q6q0500qil2a0rssai7";
+  };
+
+  sourceRoot = "source/seshat-node/native";
+
+  nativeBuildInputs = [ nodejs-14_x python3 yarn ];
+  buildInputs = [ sqlcipher ];
+
+  npm_config_nodedir = nodejs-14_x;
+
+  yarnOfflineCache = (callPackage ./yarn.nix {}).offline_cache;
+
+  buildPhase = ''
+    cd ..
+    chmod u+w . ./yarn.lock
+    export HOME=/tmp
+    yarn config --offline set yarn-offline-mirror ${yarnOfflineCache}
+    ${fixup_yarn_lock}/bin/fixup_yarn_lock yarn.lock
```

Comment:
why isn't `fixup_yarn_lock` in `nativeBuildInputs`?

Reply:
No reason, it should/could be. This is copied from GitLab, which could use the same improvement.